crates/rolldown/src/ast_scanner/mod.rs

```diff
@@ -671,4 +671,10 @@ impl<'me, 'ast: 'me> AstScanner<'me, 'ast> {
       }
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +657]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Cause: `None` maybe means the `id` is a scoped var in this function.<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/rolldown/rolldown/2657?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->